Scripts/scripts/ss_04_tor.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# standup script - Tor installation
+
+####
+# 4. Install latest stable tor
+####
+
+# Download tor
+echo "
+----------------
+  $MESSAGE_PREFIX Installing Tor
+----------------
+"
+#  To use source lines with https:// in /etc/apt/sources.list the apt-transport-https package is required. Install it with:
+if [ -z "$(which apt-transport-https)" ]; then
+  apt-get install apt-transport-https -y
```

Comment:
```suggestion
  apt-get install apt-transport-https -y
  apt-get install lsb-release apt-transport-https -y
```

Make sure the user has `lsb-release` so that setting '/etc/apt/sources.list has a valid `distribution`.